refiner-lib/src/storage.rs

```diff
@@ -90,7 +90,7 @@ fn block_hash_migration_batch(
 
     // Collect heights and hashes to migrate in this batch
     while batch.len() < BATCH_SIZE {
-        match iter.next() {
+        match iter.next().map(|r| r.unwrap()) {
```

Comment:
```suggestion
        match iter.next().map(Result::unwrap) {
```

Reply:
Thanks! We should add CI to this repo that runs clippy. Something to do in a future PR...